ci/docker-compose.yml

```diff
@@ -111,6 +111,7 @@ services:
       dockerfile: ci/Dockerfile.dev
       args:
         PYTHON_VERSION: ${PYTHON_VERSION:-3.6}
+        REQUIREMENTS_TAG: ${REQUIREMENTS_TAG}
```

Comment:
do the other backends now have there own  requirements-tag? if not I think just change all of them to have that

Reply:
I can do that, I'll probably make the requirements tag `main` for the other backends to make it concise:

- SQL, Parquet, Impala, Kudu, Clickhouse, Omnisci
  - Requirements tag will be `main`
  - Requirements file will be named `requirements-3.x-dev-main.yml`
- PySpark, Spark: 
  - Requirements tag is `pyspark-spark`
  - Requirements file is named `requirements-3.x-dev-pyspark-spark.yml`

(I want to keep `dev` in the filename because other parts of the CI have a distinction between `docs` (for the documentation) and `dev` (for the code))